server/controllers/user.js

```diff
@@ -198,12 +199,20 @@ module.exports = {
       const job = await checkStatusQueue.getJob(requestId.toString());
       job.remove();
       if (isAccepted) {
-        await Request.findByIdAndUpdate(requestId, {
+        const requestDoc = await Request.findByIdAndUpdate(requestId, {
           status: "accepted",
         });
         const reviewer = await User.findOne({
           _id: userId,
         }).select("_id name position avatar");
```

Comment:
requestDoc is not the most updated document. add {new: true} to get the most updated document

Reply:
I don't need the most updated